tests/sqlparser_postgres.rs

```diff
@@ -277,6 +277,16 @@ fn parse_create_sequence() {
         "CREATE TEMPORARY SEQUENCE IF NOT EXISTS name3 INCREMENT 1 NO MINVALUE MAXVALUE 20 OWNED BY NONE",
     );
 
+    let sql7 = "CREATE SEQUENCE name4
```

Comment:
This change didn't guard with the PostgreSQL dialect guard, so should move test case to common.

Reply:
I think it should be good to keep it inside PostgreSQL unless more dialects support this syntax.